api/mapping/views.py

```diff
@@ -1204,23 +1309,58 @@ def save_scan_report_value_concept(request):
             scan_report_value = ScanReportValue.objects.get(
                 pk=form.cleaned_data['scan_report_value_id']
             )
-
+            
             try:
-                concept = Concept.objects.get(
+                source_concept = Concept.objects.get(
                     concept_id=form.cleaned_data['concept_id']
                 )
             except Concept.DoesNotExist:
                 messages.error(request,
-                                 "Concept id {} does not exist in our database.".format(form.cleaned_data['concept_id']))
+                                 "Source Concept id {} does not exist in our database.".format(form.cleaned_data['concept_id']))
```

Comment:
This is related to a comment above. If the concept does not exist in the Concept table, we just inform the user. Later, we might want to let the user know that the concept maps to another, but it will the user responsibility to use it.